drivers/gpio/gpio_sx1509b.c

```diff
@@ -14,12 +14,6 @@
 #include <sys/byteorder.h>
```

Comment:
Please drop the changes in this file; the equivalent has been done in the updated version in topic-gpio, and doing it  here now will just create conflicts.

Reply:
I'm fine w/dropping it here, the other option is when I rebase topic-gpio we'll pick this up.  Thoughts? [i'll handle the merge conflict]

Reply:
Either way, I guess. G'head and keep it here.